examples/demo13.py

```diff
@@ -93,11 +93,11 @@ def main(argv):
     logger.debug('Read in WFIRST imaging filters.')
 
     logger.info('Reading from a parametric COSMOS catalog.')
-    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<23.5 from COSMOS.
+    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<25.2 from COSMOS.
```

Comment:
Oh I thought you meant in this demo (that the catalog was being initialized using a ```sample``` name).  I see what you mean now.